src/api/routes.rs

```diff
@@ -1,25 +1,21 @@
 use super::error::Result;
 use crate::db::Db;
-use actix_web::{
-    get, post,
-    web::{self, Json},
-    HttpResponse, Responder,
-};
+use axum::extract::State;
+use axum::{response::IntoResponse, Json};
 use serde::Deserialize;
 
-#[get("/health")]
-async fn health() -> impl Responder {
-    HttpResponse::Ok()
-}
+pub async fn health() -> impl IntoResponse {}
 
 #[derive(Debug, Deserialize)]
-struct Register {
-    address: alloy_primitives::Address,
+pub struct Register {
+    pub address: alloy_primitives::Address,
```

Comment:
why are these pub now?

Reply:
The struct itself now needs to be public because of axum , the parameter address doesn't , fixed